<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lstwState">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="lstwCommand">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="lstwAction">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="lstwPawns">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>256</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="lstwLocations">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>256</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="lstwPawn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>256</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
